test/e2e/network/conntrack.go

```diff
@@ -268,4 +269,46 @@ var _ = SIGDescribe("Conntrack", func() {
 			framework.Failf("Failed to connect to backend 2")
 		}
 	})
+
+	ginkgo.AfterEach(func() {
```

Comment:
Is it possible that AfterEach is too late?
Should we consider doing e.g. periodically?

Reply:
it should be enough, udp timeout for new connectins is 30sec, but if the connection "sees" more packets it extended the timeout to 120sec, so if the conntrack deletion does not flush the stale entries we should see it there
xref https://www.kernel.org/doc/Documentation/networking/nf_conntrack-sysctl.txt
```
nf_conntrack_udp_timeout - INTEGER (seconds)
	default 30

nf_conntrack_udp_timeout_stream - INTEGER (seconds)
	default 120

	This extended timeout will be used in case there is an UDP stream
	detected.
```

Reply:
I don't have strong opinion here, but if we are unlucky and are missing couple seconds, that different may matter. But we may also start with what you have and if that won't help iterate further.

Reply:
yeah, let's try to hit first one  failure in this PR before merging

Reply:
I'm going to run it with a temp commit to see the results, if is ok I'd like to merge because seems is hard to catch a failure in this PR

Reply:
Yes - let's merge.
Though, please add a TODO with the issue number to get rid of it once this is debugged and fixed.